src/home_work_14_1.ts

```diff
@@ -0,0 +1,96 @@
+interface Note {
+  id: string;
+  title: string;
+  content: string;
+  createdAt: Date;
+  updatedAt: Date;
+  status: "completed" | "pending";
```

Comment:
It would be better to create an enum for these cases. Using string values in code is not a best practice.